contribute/drone-pipeline.md

```diff
@@ -1,17 +1,17 @@
 # Making changes to the Drone pipeline
 
-_Only people in the Grafana organization can make changes to the Drone pipeline_
+> Only members of the Grafana Organization can make changes to the Drone pipeline.
```

Comment:
Judging from the GitHub documentation, they do not capitalize "organization".

Reply:
```suggestion
> Only members of the Grafana organization can make changes to the Drone pipeline.
```